src/locale/locales/fr/messages.po

```diff
@@ -9,7 +9,7 @@ msgstr ""
 "Project-Id-Version: \n"
 "Report-Msgid-Bugs-To: \n"
 "PO-Revision-Date: 2024-11-09 19:42+0100\n"
-"Last-Translator: Stanislas Signoud (@signez.fr)\n"
+"Last-Translator: Cereza Auclair (@cereza.zone)\n"
 "Language-Team: Stanislas Signoud (@signez.fr), surfdude29\n"
```

Comment:
I concur: if you are okay with that, please do @CerryTsuki!

Reply:
i was reluctant to put it there because i can't guarantee i'll be consistently contributing, but if both of you are okay with it i'll put it back in

Reply:
This shouldn't be read as a commitment :)

(Also, we had no problem working on translations before, so maybe it'll be just about spotting some weird things we'll overlook on our next PR! There are also talks in #2940 of maybe moving to a crowd-translating platform later so it could become even easier)